.github/workflows/build.yml

```diff
@@ -146,3 +146,32 @@ jobs:
       - run: ./bootstrap
       - run: scripts/install_cppcheck.sh $CPPCHECK_REPO $CPPCHECK_VER
       - run: scripts/run_cppcheck.sh -v $CPPCHECK_VER
+
+  code_formatting_check:
+    name: code formatting check
+    runs-on: ubuntu-latest
```

Comment:
This might be unstable because the "latest" version will change as time flows but not a big problem.